CoronaCare/src/navigation/NavigationTabInitialiser.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import { Navigation } from "react-native-navigation"
+
+const NavigationTabInitialiser = () => {
+    Navigation.setRoot({
+        root: {
+            bottomTabs: {
+                children: [{
+                    stack: {
+                        children: [{
+                            component: {
+                                name: 'navigation.CoronaCare.RecordsScreen',
+                                passProps: {
+                                    text: 'This is tab 1'
```

Comment:
why is it so nested

Reply:
The Navigation library works in a nested fashion. So a stack has multiple components with the root being the first component to load up. 

In this example: 

`bottomTabs`: loads up a bottomTabs instance and within each tab, it needs to load a screen
`children`:  defines all the child components of the stack (ie. the screens to be shown on each tab)

I am also not very experienced with this, I am pretty sure we can cut down the nesting by reading up on the documentation and figuring out the best way to do it.